<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Aucom</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Original">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Original Audio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dubbed Audio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Original">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Dub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Original">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>18</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Browse original audio file</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dub">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>58</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Browse dubbed audio file</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Compare">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>18</y>
      <width>80</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Import audio files and compare them to find gaps </string>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gaps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>List of founded gaps:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Gaps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Play specific part of original audio file</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Progress">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_parameters"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionOriginal"/>
     <addaction name="actionDub"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="action_OriginalAudio"/>
    <addaction name="action_DubbedAudio"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionOriginal_Audio"/>
    <addaction name="actionDubbed_Audio"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuPlay"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Audio_files">
   <property name="text">
    <string>Import audio files</string>
   </property>
  </action>
  <action name="actionOriginal_Sound_Wave">
   <property name="text">
    <string>Original Sound Wave</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionDub">
   <property name="text">
    <string>Dub</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="statusTip">
    <string>Export list of founded gaps</string>
   </property>
  </action>
  <action name="action_OriginalAudio">
   <property name="text">
    <string>Original audio</string>
   </property>
  </action>
  <action name="action_DubbedAudio">
   <property name="text">
    <string>Dubbed audio</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionOriginal_Audio">
   <property name="text">
    <string>Original audio</string>
   </property>
  </action>
  <action name="actionDubbed_Audio">
   <property name="text">
    <string>Dubbed audio</string>
   </property>
  </action>
  <action name="actionSet_parameters">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
